app/frontend/src/features/messages/groupchats/GroupChatsTab.stories.tsx

```diff
@@ -1,33 +1,11 @@
-import * as React from "react";
-// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Meta, Story } from "@storybook/react/types-6-0";
+import * as React from "react";
 import { Provider } from "react-redux";
+import { message1 } from "../../../__mocks__/service";
```

Comment:
Didn't you update the Webpack alias config in the Storybook so you can import it like this?

```javascript
import { message1 } from "../../../service";
```
Or does that not actually work?

Reply:
I think it will work. But my imports are auto-generated and if my IDE finds only one possibility it does not show the import line to me in advance. So I missed this. It would be even nicer if we change tsconfig.json in such a way we can do just `import service from 'service';`.

Reply:
But in the editor typescript complains about it.

Reply:
Ahh yes I noticed this too. I don't think this is possible without using something like [`react-app-rewired`](https://www.npmjs.com/package/react-app-rewired), since Create React App is quite restrictive with what alias you can set...